.gitignore

```diff
@@ -7,3 +7,5 @@ dist
 .vscode
 
 assets/i18n/translate.*.json
+
+public/jaas
```

Comment:
You are adding this folder in `.gitignore`, but you are also serving files from this folder. So, if I clone this repo, will creating the build of this plugin also generate this folder inside my public directory?

Reply:
yes make dist will generate folder named Jaas inside your public directory